dask_geopandas/core.py

```diff
@@ -343,6 +344,38 @@ def hilbert_distance(self, p=15):
 
         return distances
 
+    def morton_distance(self, p=15):
+
+        """
+        Calculate distance of geoms along Morton curve
```

Comment:
```suggestion
        Calculate the distance of geometries along the Morton curve
        
        The Morton distance can be used to spatially partition Dask-GeoPandas objects,
        by mapping two-dimensional geometries along the Morton space-filing curve.
        
        Each geometry is represented by the midpoint of its bounds and linked to the 
        Morton curve. The function returns a distance from the beginning 
        of the curve to the linked point.
        
        Morton distance is more performant than ``hilbert_distance`` but can result in 
        less optimal partitioning.
```

It is better to have all info in the main docstring.

Reply:
thank you